scipy/constants/codata.py

```diff
@@ -52,7 +52,7 @@
 from __future__ import division, print_function, absolute_import
 
 import warnings
-from math import pi, sqrt
+from math import pi
```

Comment:
sqrt symbol will be needed again in gh-11345, so I left it in and deleted gh-11537.

Reply:
OK, I've removed this change in my latest update.